logever/src/main/java/ai/smartfac/logever/controller/UserController.java

```diff
@@ -35,11 +39,15 @@ public ResponseEntity<?> getUsers(@PathVariable("username") String username) {
     public ResponseEntity<?> registerUser(@RequestBody User user) {
         Optional<User> existingUser = userService.getUserByUsername(user.getUsername());
         User newUser;
-        if(existingUser.isEmpty())
+        if(existingUser.isEmpty()) {
+            user.setPassword(bCryptPasswordEncoder.encode(user.getPassword()));
             newUser = userService.saveUser(user);
+        }
         else {
```

Comment:
I think we should just throw error when a user is already present. In future, we should have a PUT api to update the user.

Reply:
Done!